bridges/GoogleScholarBridge.php

```diff
@@ -1,20 +1,102 @@
 <?php
 
-class GoogleScholarBridge extends BridgeAbstract
+class GoogleScholarV2Bridge extends BridgeAbstract
 {
-    const NAME = 'Goolge Scholar';
+    const NAME = 'Google Scholar v2';
     const URI = 'https://scholar.google.com/';
-    const DESCRIPTION = 'Follow authors of scientific publications.';
-    const MAINTAINER = 'thefranke';
+    const DESCRIPTION = 'Search for publications or follow authors on Google Scholar.';
+    const MAINTAINER = 'nicholasmccarthy@github';
     const CACHE_TIMEOUT = 86400; // 24h
 
-    const PARAMETERS = [[
-        'userId' => [
-            'name' => 'User ID',
-            'exampleValue' => 'qc6CJjYAAAAJ',
-            'required' => true
-        ]
-    ]];
+    const PARAMETERS = [
+        'user' => [
+            'userId' => [
```

Comment:
@dvikan Will this reordering break subscriptions?

Reply:
i dont think so. will only reorder the url get params

Reply:
And the parameter is still `userId` even if its now in a group?

Reply:
sorry im not sure

Reply:
I'm not 100% sure, but I manually tested that it was still working with just the userId bridge set, and after the switch for context change the code is unchanged from previous so I think it should not break anything. _Think_.

Reply:
OK, with the v2 bridge installed side by side with the original, setting the userId in each produces the following URLs:

v1: `http://rss-bridge/?action=display&bridge=GoogleScholarBridge&userId=uTMgpiAAAAAJ&format=Html`
v2: `http://rss-bridge/?action=display&bridge=GoogleScholarV2Bridge&context=user&userId=uTMgpiAAAAAJ&format=Html`

Removing the ```&context=user``` from the v2 URL gives: 
`http://rss-bridge/?action=display&bridge=GoogleScholarV2Bridge&userId=uTMgpiAAAAAJ&format=Html`

which is still still creating a valid feed, so unless I'm very much mistaken this is not a breaking change for any existing feeds.

Reply:
Amazing, in that case everything keeps working an we got loads more options. Thanks for the changes!